tests/e2e/app-files/main.R

```diff
@@ -1,16 +1,20 @@
-box::use(shiny)
-
-box::use(app / view / hello)
-
-#' @export
-ui <- function(id) {
-  ns <- shiny$NS(id)
-  hello$ui(ns("hello"))
-}
-
-#' @export
-server <- function(id) {
-  shiny$moduleServer(id, function(input, output, session) {
-    hello$server("hello")
-  })
-}
+box::use(
+  shiny,
+  rhino[log]
+)
+
+box::use(app / view / hello)
+
+#' @export
+ui <- function(id) {
+  ns <- shiny$NS(id)
+  hello$ui(ns("hello"))
+}
+
+#' @export
+server <- function(id) {
+  shiny$moduleServer(id, function(input, output, session) {
+    log$trace("This is a test")
```

Comment:
I only tested trace to keep it minimal, but I can add the other log functions too.